### ISSUE TITLE
Extension installs but just launches an external terminal window

### ISSUE THREAD

**Issue by deedan** [artoale/brackets-terminal#39]
I was under the impression that the extension opens a terminal window inside Brackets that can share the workspace with other windows displaying files (e.g. tiled vertically / horizontally etc).

I installed the extension on my iMac running Yosemite 10.10.5.  I see a terminal icon in two places - one is on the left side extension toolbar.  

![image](https://cloud.githubusercontent.com/assets/17747305/13641395/406236c8-e5e6-11e5-849d-51d1819fb419.png)

The second location is on the toolbar over the right side of the Brackets Git extension:

![image](https://cloud.githubusercontent.com/assets/17747305/13641421/5a5dedf6-e5e6-11e5-9007-056b00383047.png)

Clicking either button launches a standalone terminal window.  

So in short I can't get the brackets-terminal extension to work as described.   Has anyone else had any luck, and if so, how did you install and/or troubleshoot?

**Comment by Jeff-Russ** [comment]
I added the directory "Install-brackets-terminal" in my fork of this repo, which contains a Mac OS installer. My solution is a bit hacky (it actually modifies the name of the application to hide it, putting a dummy in it's place) but it worked for me. If you are wary about running an installer: it's really just a bash script. Take a look at the script and you can pull of what I did or modify it to your needs. https://github.com/Jeff-Russ/brackets-terminal/tree/master/Install-brackets-terminal
